docs/ref/class-based-views/mixins-single-object.txt

```diff
@@ -66,7 +66,7 @@ Single object mixins
         non-sequential arguments. Using a unique slug may serve the same
         purpose, but this scheme allows you to have non-unique slugs.
 
-        .. _insecure direct object reference: https://www.owasp.org/index.php/Top_10_2013-A4-Insecure_Direct_Object_References
+        .. _insecure direct object reference: https://owasp.org/www-project-top-ten/
```

Comment:
I tested  https://www.owasp.org/index.php/Top_10_2013-A4-Insecure_Direct_Object_References and it's redirected to https://owasp.org/www-project-top-ten/ . Don't know is it the right page or not.

Reply:
https://wiki.owasp.org/index.php/Top_10_2013-A4-Insecure_Direct_Object_References

Reply:
Yeah, that is the 2013 top ten. It got merged into `A5:2017-Broken Access Control`, see https://owasp.org/www-project-top-ten/OWASP_Top_Ten_2017/Top_10-2017_Release_Notes so this would mean that the link text might also need be updated. Alternatively, you could point the link to https://cheatsheetseries.owasp.org/cheatsheets/Insecure_Direct_Object_Reference_Prevention_Cheat_Sheet.html and in that case you can keep the 'insecure direct object reference' name.